chat.delta.desktop.yml

```diff
@@ -23,6 +23,8 @@ finish-args:
   - --share=network
   - --talk-name=org.freedesktop.Notifications # needed for dbus notification, as the notification portal crashed because of a bug in electron https://github.com/electron/electron/issues/40461
   - --talk-name=com.canonical.AppMenu.Registrar # fix https://github.com/flathub/chat.delta.desktop/issues/118
+  - --talk-name=org.kde.StatusNotifierWatcher # fix https://github.com/flathub/chat.delta.desktop/issues/143
```

Comment:
Looking at https://docs.flatpak.org/de/latest/desktop-integration.html it says:
"Most implementations of StatusNotifier have dropped this requirement, but known exceptions include Electron versions older than 23.3.0."
and "This introduces many new risks, including the ability to impersonate any KDE service or application, possibly capturing important user data."
We use electron 37.1
Can you explain why this is used?